app/assets/stylesheets/map.scss

```diff
@@ -0,0 +1,8 @@
+#map-container {
+   height: 400px;
+   margin-top: 25px;
+ }
+ #map-canvas {
```

Comment:
Avoid using id selectors<br>Line should be indented 0 spaces, but was indented 1 spaces